test/test_meta.py

```diff
@@ -1303,6 +1303,32 @@ def test_map_location_deserialize(self):
         self.assertEqual(r.dtype, t.dtype)
         self.assertEqual(r.storage().data_ptr(), 0)
 
+    def test_embedding_bag_byte_prepack(self):
+        batch_size = 10
+        num_embeddings = 80
+        embedding_dim = [128, 256, 512]
+        res_shape = [[batch_size, num_embeddings, ed + 8] for ed in embedding_dim]
+        for ed, rs in zip(embedding_dim, res_shape):
```

Comment:
a better test approach is using the cpu backend to run and compare the shape.

Reply:
lol I tried that in this test_meta.py, but it does not allow me to use cpu backend. Instead, I used the op bench for two backends to compare the shape.